Gemfile

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
```

Comment:
I'm not sure if it matters either way, but do we need this? I don't see it in any of our other plugins.

Reply:
yeah looks a bit odd.

Reply:
That's the current template from `bundle init`. I'm removing the line.